packages/libffi/CONFIG

```diff
@@ -0,0 +1,25 @@
+libffi_VERSION ?= 3.3-rc0
```

Comment:
Why version 3.3, particularly as we only have an rc0 candidate?  I'm guessing there's a reason 3.2.1 isn't suitable.

Reply:
I assumed the patches were made for that version, but if you want, I could try  rc2.
Regarding version 3.2.1, it's more than 5 years old, so I guess there are major changes that it's worth getting.